### ISSUE TITLE
Platform table - aria-rowindex should not start with 0 value

### ISSUE THREAD

**Issue by esporysz** [SAP/fundamental-ngx#12778]
#### Is this a bug, enhancement, or feature request?

bug

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)

angular: 18.2.4
fundamental: 0.53.1-rc.23

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;

1. Crete some platform table. It may be copy of [example from documentation](https://sap.github.io/fundamental-ngx/#/platform/table/preserved-state).
2. `aria-rowindex` starts with 0 value according to [documentation](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-rowindex) it should be an integer greater or equal to 1.
